cprover_bindings/src/goto_program/builtin.rs

```diff
@@ -17,6 +17,8 @@ pub enum BuiltinFn {
     Copysignf,
     Cos,
     Cosf,
+    Error,
+    ErrorNoLocation,
```

Comment:
There is no comment at the top of this enum documenting it. Is this the list of standard function supported by CPROVER? Can I add any builtin function from GCC that CBMC supports as well? Like anything from [here](https://github.com/diffblue/cbmc/tree/71d07e0d507b31220379f54eefb1bf3bb0f7a184/src/ansi-c/compiler_headers).

Reply:
Yes. You can also see that this model doesn't scale really well. The other problem is that the Rust `libc` signature doesn't always matches CBMC's (although they have the same ABI).